docs/Maintainer-Guidelines.md

```diff
@@ -104,3 +104,14 @@ Whitespace corrections (to Ruby standard etc.) are allowed (in fact this
 is a good opportunity to do it) provided the line itself has some kind
 of modification that is not whitespace in it. But be careful about
 making changes to inline patches—make sure they still apply.
+
+## Communication
+Maintainers have a variety of ways to communicate with each other:
+
+- Homebrew's public repositories on GitHub
+- Homebrew's Slack/Discourse/GitHub repositories/other private communications
```

Comment:
The repetition between this line and the line that follows is confusing.

Reply:
@ilovezfs Yeh; have reordered it a bit to make it a bit clearer